File: server/zlj2019/game_server/src/com/imop/lj/gameserver/pet/msg/CGPetHorseChangeName.java
package com.imop.lj.gameserver.pet.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.pet.handler.PetHandlerFactory;

/**
 * 骑宠改名
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPetHorseChangeName extends CGMessage{
	
	/** 骑宠唯一Id */
	private long petId;
	/** 名字 */
	private String newName;
	
	public CGPetHorseChangeName (){
	}
	
	public CGPetHorseChangeName (
			long petId,
			String newName ){
			this.petId = petId;
			this.newName = newName;
	}
	
	@Override
	protected boolean readImpl() {

	// 骑宠唯一Id
	long _petId = readLong();
	//end


	// 名字
	String _newName = readString();
	//end



			this.petId = _petId;
			this.newName = _newName;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 骑宠唯一Id
	writeLong(petId);


	// 名字
	writeString(newName);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_PET_HORSE_CHANGE_NAME;
	}
	
	@Override
	public String getTypeName() {
		return "CG_PET_HORSE_CHANGE_NAME";
	}

	public long getPetId(){
		return petId;
	}
		
	public void setPetId(long petId){
		this.petId = petId;
	}

	public String getNewName(){
		return newName;
	}
		
	public void setNewName(String newName){
		this.newName = newName;
	}


	@Override
	public void execute() {
		PetHandlerFactory.getHandler().handlePetHorseChangeName(this.getSession().getPlayer(), this);
	}
}

File: u3u2/Assets/Scripts/app/net/c2s/CGPetHorseChangeName.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 骑宠改名
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPetHorseChangeName :BaseMessage
{
	
	/** 骑宠唯一Id */
	private long petId;
	/** 名字 */
	private string newName;
	
	public CGPetHorseChangeName ()
	{
	}
	
	public CGPetHorseChangeName (
			long petId,
			string newName )
	{
			this.petId = petId;
			this.newName = newName;
	}
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{
	// 骑宠唯一Id
	WriteLong(petId);
	// 名字
	WriteString(newName);

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_PET_HORSE_CHANGE_NAME;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}